Update light client tutorial (#1902)
substrate-developer-hub/substrate-docs: content/md/en/docs/tutorials/integrate-with-tools/integrate-a-light-client-node.md

As you learned in [Light clients in Substrate Connect](/fundamentals/light-clients-in-substrate-connect/), light client nodes provide secure and decentralized access to blockchain data with minimal hardware and software requirements.

This tutorial demonstrates how you can connect to any Substrate-based blockchain using a light client.
To illustrate this, you'll learn how to connect your application to the [Statemint parachain](https://wiki.polkadot.network/docs/learn-statemint).

> Statemint is a common good parachain that is connected to Polkadot and has a publicly-accessible chain specification file.

Reviewer: ```suggestion
Statemint is a system parachain that is connected to Polkadot and has a publicly-accessible chain specification file.
```

## Before you begin

Before you begin, verify the following:

> - You have a code editor set up for working with Javascript and Typescript

Reviewer: ```suggestion
- You have a code editor set up for working with Javascript and Typescript.
```

> - You have the [`yarn` package manager](https://classic.yarnpkg.com/lang/en/docs/install/) installed

Reviewer: ```suggestion
- You have the [`yarn` package manager](https://classic.yarnpkg.com/lang/en/docs/install/) installed.
```

   cd empty-webapp
   ```
   

> 4. Install Substrate Connect

Reviewer: ```suggestion
4. Install Substrate Connect by running the following command:
```
